ignite/services/network/join.go

```diff
@@ -85,16 +85,70 @@ func (n Network) Join(
 	}
 
 	if !o.accountAmount.IsZero() {
-		if err := n.sendAccountRequest(ctx, launchID, accountAddress, o.accountAmount); err != nil {
+		if err := n.SendAccountRequest(ctx, launchID, accountAddress, o.accountAmount); err != nil {
 			return err
 		}
 	}
 
-	return n.sendValidatorRequest(ctx, launchID, peer, accountAddress, gentx, gentxInfo)
+	return n.SendValidatorRequest(ctx, launchID, peer, accountAddress, gentx, gentxInfo)
 }
 
-// sendValidatorRequest creates the RequestAddValidator message into the SPN
-func (n Network) sendValidatorRequest(
+func (n Network) SendAccountRequestForCoordinator(ctx context.Context, launchID uint64, amount sdk.Coins) error {
+	addr, err := n.account.Address(networktypes.SPN)
+	if err != nil {
+		return err
+	}
+
+	return n.SendAccountRequest(ctx, launchID, addr, amount)
+}
+
+// SendAccountRequest creates an add AddAccount request message.
+func (n Network) SendAccountRequest(
+	ctx context.Context,
+	launchID uint64,
+	address string,
+	amount sdk.Coins,
+) error {
+	addr, err := n.account.Address(networktypes.SPN)
+	if err != nil {
+		return err
+	}
+
+	msg := launchtypes.NewMsgSendRequest(
+		addr,
+		launchID,
+		launchtypes.NewGenesisAccount(
+			launchID,
+			address,
+			amount,
+		),
+	)
+
+	n.ev.Send("Broadcasting account transactions", events.ProgressStarted())
```

Comment:
This makes me think that maybe we should settle on a standard for spinner status texts. In other places texts use the "..." at the end to express that something is happening and might take some time to complete, that is what I understand, which might be a legacy "standard" coming from before the spinner; Now, the spinning icon is the one expressing what the "..." was doing before.

I think we should remove the "..." in favor of the spinner.

Just wanted to comment on that.

Reply:
I may think there is no `...` added in this PR but it is a good point, let's open an issue for this

Reply:
Yes, there are no dots in this one. I will open an issue then.